ITC205_BaseLine_Code/src/ReturnBookControl.java

```diff
@@ -1,76 +1,76 @@
+//Luke: Ready for static review
+

```

Comment:
D25  - word wrap on lines 21 31 61 69    in Return Book Control

Reply:
Confirmed Fixed - Facilitator